styles/colors-gallium-day.less

```diff
@@ -66,11 +66,6 @@
 @agate-colorpicker-bg-color: rgba(0, 0, 0, 0.5);
 @agate-colorpicker-palette-shadow: @agate-popup-shadow;
 
-// Divider
-// ---------------------------------------
-@agate-divider-color:        @agate-text-color;
```

Comment:
It doesn't look like there is a new `@agate-heading-color` here, also there are 2 definitions for `@agate-heading-border-color` below (line 83 and 86)

Same thing in `colors-gallium-night.less`